src/HPolyhedron.jl

```diff
@@ -347,6 +349,34 @@ end
     remove_redundant_constraints(P::PT;
                                  backend=GLPKSolverLP()) where {N, HPoly{N}}
 
+Given a polyhedron in H-representation, return a new polyhedron with no reundant
+constraints.
+
+## Input
+
+- `P`       -- polyhedron
+- `backend` -- (optional, default: `GLPKSolverLP`) the numeric LP solver backend
+
+## Output
+
+The polyhedron obtained by removing the redundant constraints in `P`.
+
+## Algorithm
+
+See `remove_redundant_constraints!`. 
+"""
+function remove_redundant_constraints(P::PT;
+                                      backend=GLPKSolverLP()) where {N, PT<:HPoly{N}}
+    return remove_redundant_constraints!(copy(P), backend=backend)
```

Comment:
I am not sure, but I guess that `copy(P)` does not copy the constraints because we never override `copy`. Here is what the docs say:
```
copy(x)

  Create a shallow copy of x: the outer structure is copied, but not all internal values.
```
Can you check? If that is true, you need to copy the array of constraints instead and create a new `PT` with it. And we should create an issue to override `copy`.

Reply:
Sorry, I just saw that you override `copy` already :sob: